ztp/gitops-subscriptions/argocd/example/siteconfig/example-sno.yaml

```diff
@@ -14,6 +14,11 @@ spec:
   clusters:
   - clusterName: "example-sno"
     networkType: "OVNKubernetes"
+    # Optionally; installConfigOverrides is a generic way of passing install-config
+    # parameters through the siteConfig.  The capabilities field configures the
+    # composable openshift feature .  In this example, we remove all but the
+    # marketplace component from the optional set of components.
+    #installConfigOverrides:  "{\"capabilities\":{\"baselineCapabilitySet\": \"None\", \"additionalEnabledCapabilities\": [ \"marketplace\" ] }}"
```

Comment:
For the primary use case is this optional, or do we expect new clusters to include this setting? If this is intended as part of the reference configuration we should uncomment it here. @browsell what do you think?

Reply:
This needs to be part of the reference config, not optional

Reply:
Removed optional tag and comment